### ISSUE TITLE
Record from webcam?

### ISSUE THREAD

**Issue by orschiro** [green-project/green-recorder#34]
Dear all, 

Yesterday, I had the need to record myself for a video using my webcam and realised I couldn't with green-recorder. 

I think this would be a really cool feature. 

What do others think?

**Comment by mhsabbagh** [comment]
Yep. It would be nice to go with. ffmpeg supports it. We just need to deal with the recording devices to use. I still couldn't manage to manipulate those devices (how to list the possible input audio/video devices and how to choose the real working on).

**Comment by orschiro** [comment]
Maybe [Vokoscreen](https://github.com/vkohaupt/vokoscreen) can be of inspiration here?

They support webcams.

**Comment by titaniumbones** [comment]
Hey there, I'm just wondering if there's any chance of movement on this feature? I'm unfortunately not in a position to contribute code for at least the next few weeks, but if you are interested in making a push, I could try to help a little.  You mention having trouble figuring out hte devices -- perhaps if you are able to break down the issue into its ocmponents I could try to helpwith some small part.  

Thank you for this great tool!

**Comment by Dharni0607** [comment]
I would like to work on this issue, if nobody has taken it up. Please let me know if I can continue.
